packages/core/lib/flutter_generator.dart

```diff
@@ -13,13 +14,15 @@ class FlutterGenerator {
     this.pubspecFile, {
     this.buildFile,
     this.assetsName = 'assets.gen.dart',
+    this.shadersName = 'shaders.gen.dart',
```

Comment:
Could you place shades after fonts so they are sorted alphabetically? Also the import the below predicate, and other files